metaflow/plugins/kfp/kfp.py

```diff
@@ -105,6 +106,56 @@ def _get_retries(node):
 
         return max_user_code_retries, max_user_code_retries + max_error_retries
 
+    @staticmethod
+    def _get_resource_requirements(node):
```

Comment:
Are you planning to introduce a Kubernetes integration as well which will allow folks to prototype their workflows directly through the Metaflow scheduler?

Reply:
Our use case is to integrate with Kubeflow (KFP), to support Kubernetes without Kubeflow/KFP is not in our plans.

Reply:
I don't think we are planning to integrate with Kubernetes directly. Meanwhile, KFP is basically using Kubernetes interface for resource managements. Some code or design here can be a good starting point. Feel free to suggest designs that helps future Kubernetes integration.

Reply:
It would be great if a user could use `--with:kfp` or `--with:kubernetes` just as they can use `--with:batch`, while `kfp` (and maybe future Argo support) is just the scheduler (like step functions).

That'd be great to support.  But we are first supporting to run on KFP...